pkgs/development/python-modules/plum-py/default.nix

```
@@ -9,7 +9,7 @@
 
 buildPythonPackage rec {
   pname = "plum-py";
-  version = "0.8.6";
+  version = "0.8.7";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and use `build-system`